code/datums/station_traits/negative_traits.dm

```diff
@@ -81,7 +81,7 @@
 
 /datum/station_trait/slow_shuttle
 	name = "Slow Shuttle"
-	trait_type = STATION_TRAIT_NEUTRAL
+	trait_type = STATION_TRAIT_NEGATIVE
```

Comment:
Undocumented balance change? 🙂

Reply:
it isn't undocumented (unless you mean in the changelog)

> This PR also fixes a minor error in which "Slow shuttle" station trait was classified as neutral instead of negative. (It's in the negative file and it's a negative trait, it appears to be an typo that it's neutral.)

Reply:
Yeah, I meant the changelog, but for future reference these should be in separate PRs.

Reply:
Fair enough, it just annoyed me 'cause I was looking for where to put negative traits and saw a neutral trait at the bottom of the negative file.